### ISSUE TITLE
healthcheck fails if apple id has underscore in it

### ISSUE THREAD

**Issue by SoulOfNoob** [boredazfcuk/docker-icloudpd#27]
example: 
my apple id format is: `firstname_lastname@provider.com`
generated cookie file is: `firstname_lastnameprovidercom`
healthcheck.sh expects `firstnamelastnameprovidercom` (i checked with a print statement)
therefore the healthcheck fails.

manually renaming the cookie file doesn't help either because it just gets regenerated.

**Comment by SoulOfNoob** [comment]
local quick fix for me: changed regex to include underscore
`sync-icloud.sh:13` and `healthcheck.sh:19`:
changed `${apple_id//[^a-zA-Z0-9]/}` to `${apple_id//[^a-zA-Z0-9_]/}`

because i could not find the reason why the underscore is not filtered by file creation

**Comment by boredazfcuk** [comment]
Hmmmm... This sounds a bit odd to be honest.

I used to have an underscore in my Apple ID, so this did work once upon a time. I had to swap to a non-underscored ID when I setup family safety on my older devices as Apple don't support underscores in the e-mail address used for parent accounts...

However, main sync-icloud.sh script also uses the same parameter expansion to calculate the cookie's filename (line 13).

This is used a lot throughout the script, so if you have an underscore in your address, I'd expect the main script to bomb out too.

Have you changed line 13 of the main script too?
